Mars_Application_UserStory2_TestAutomation/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Microsoft.VisualBasic.FileIO;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Assert = Xunit.Assert;
+
+namespace Mars_Application_UserStory2_TestAutomation.Pages
+{
+    public class ProfilePage
+    {
+        public void CreateLanguage(IWebDriver driver, string language, string languageLevel)
+        {
+            Thread.Sleep(1000);
+            IWebElement AddnewButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
```

Comment:
you need to implement the POM pattern please https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html 
also fix this for all pages